website/docs/faqs/Git/gitignore.md

```diff
@@ -1,25 +1,224 @@
+3. Open the new or existing '.gitignore` file, and add the following entries.  Here is a link to a [sample dbt .gitignore file.](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore).
+> NOTE: These can go anywhere in the file, as long as they are on separate lines.  The lines as shown are wildcards that will include all nested files and folders. Do NOT append a trailing '*'.  See [these git docs](https://git-scm.com/docs/gitignore) for more information on `.gitignore` syntax.
+```bash
+target/
+dbt_packages/
+logs/
+# legacy -- renamed to dbt_packages in dbt v1
+dbt_modules/
+```
+3. Save the changes but _don't commit_.
+4. Restart the IDE by clicking on the three dots next to the **IDE Status button** on the lower right corner of the IDE screen, then selecting **Restart IDE**.
 
-4. Go back into the Cloud IDE and reclone your repository. This can be done by clicking on the green "ready" in the bottom right corner of the IDE (next to the command bar), and then clicking the orange "reclone repo" button in the pop up.
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/restart-ide.jpg" width="50%" title="Restart the IDE by clicking the three dots on the lower right or click on the Status bar" />
+
+5. Once the IDE restarts, delete the following files or folders from the file explorer pane (if they exist).  No data will be lost.:
+    * `target`, `dbt_modules`, `dbt_packages`, `logs`
+6. **Save** and then **Commit and sync** the changes.
+7. Restart the IDE again using the same procedure as step 3.
+8. Once the IDE restarts, use the 'Create a pull request' (PR) button under the **Version Control** menu to start the process of integrating the changes.
+9. When the git provider's website opens to a page with the new PR, follow the necessary steps to compelete and merge the PR into the main branch of that repository.
+> NOTE: The 'main' branch might also be called 'master', 'dev', 'qa', 'prod' or something else depending on the organizational naming conventions.  The goal is to merge these changes into the root branch that all other development branches are created from.
+10. Return to the dbt Cloud IDE.
+11. Use the 'Change Branch' button, to switch to the main branch of the project.
+12. Once the branch has changed, click the **Pull from remote** button to pull in all the changes. You can verify the changes by making sure the files/folders in the .gitignore file are in italics. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/gitignore-italics.jpg" width="50%" title="A dbt project on the main branch that has properly configured gitignore folders (highlighted in italics)."/>
+
+### Steps to use the git providers web interface to fix a broken .gitignore:
+   
+Sometimes it is necessary to use the git providers web interface to fix a broken `.gitignore` file. While the exact sequence of clicks will be different for each provider, the overall steps are the same.
```

Comment:
hey @will-sargent-dbtlabs, why would it be necessary to use the git providers ui to fix the file?